### PR TITLE
Moving existing environment docs to precise build image page

### PR DESCRIPTION
**Pull request by kimh**
We now have two Linux build images where different things/versions are installed and just a single `environment` page doesn't work anymore. So, creating build image page for precise and trusty and makes pointers to each pages from necessary places.

### CODE REVIEW

**Comment by kimh** [comment]
wow, we have tests!

**Comment by kimh** [comment]
ping

File: jekyll/_docs/android.md [hunk 1]
```diff
@@ -101,7 +100,7 @@ test:
 ```
 
 `circleci-android22` is an AVD preinstalled on the machine for Android 22 on the ARM V7 EABI.
-There's also a corresponding `circleci-android21`; alternatively, you can 
+There's also a corresponding `circleci-android21`; alternatively, you can
```
Comment (z00b) [inline jekyll/_docs/android.md]: 'can' is duplicated with the text inside the link.

File: jekyll/_docs/android.md [hunk 2]
```diff
@@ -16,11 +16,7 @@ CircleCI supports building and testing Android applications.
 The SDK is already installed on the VM at `/usr/local/android-sdk-linux`. We export
 this path as `$ANDROID_HOME`.
 
-We have the following SDK packages preinstalled:
-
-{% for version in site.data.versions.android_sdk_packages %}
-- `{{ version }}`
-{% endfor %}
+We have many versions of Android SDK pre-installed on [Ubuntu 12.04]({{ site.baseurl }}/build-image-precise/#android-sdk) and [Ubuntu 14.04]({{ site.baseurl }}/build-image-trusty/#android-sdk) build images.
```
Comment (z00b) [inline jekyll/_docs/android.md]: "We have many versions of *the* Android SDK..."

**Comment by z00b** [comment]
I don't really know how the docs work, so I can't spot errors in the generation code, but I'd ship it and fix any issues in production.

File: jekyll/_docs/configuration.md [hunk 3]
```diff
@@ -191,8 +191,8 @@ hyphens (-) and dots (.).
 CircleCI uses [RVM](https://rvm.io/) to manage Ruby versions.
 We use the Ruby version you specify in your `.rvmrc`, your
 `.ruby-version` file, or your Gemfile.
-If you don't have one of these files, we'll use Ruby `{{ site.data.versions.default_ruby }}`
-or `{{ site.data.versions.old_ruby }}`, whichever we think is better.
+If you don't have one of these files, we'll use Ruby `{{ site.data.precise.versions.default_ruby }}`
+or `{{ site.data.precise.versions.old_ruby }}`, whichever we think is better.
```
Comment (eric-hu) [inline jekyll/_docs/configuration.md]: This maps out to 1.8.7-p358.  Do we still use that?

Also, this refers just to Precise.  Should this section also mention Trusty?

**Comment by eric-hu** [comment]
Cloned and checked the generated pages locally.  I made a commit to change the grammar in one place.

I think there's a bit of a combinatorial explosion with the version listings and the split into Precise and Trusty pages.  However, I think that's a separate issue.

I'm very much in favor of version lists for Precise and Trusty, so LGTM.

**Comment by kimh** [comment]
:bomb:

**Comment by felicianotech** [comment]
I didn't have a chance to fully review this PR before it got merged. This introduced some issues that I've documented in #70

**Pull request merged**